tinodesdk/src/main/java/co/tinode/tinodesdk/Tinode.java

```diff
@@ -773,7 +793,10 @@ public void oobNotification(Map<String, String> data, String authToken, boolean
                 }
 
                 if (topic.getSeq() < seq) {
-                    boolean newConnection = syncLogin(authToken);
+                    if (!syncLogin(authToken)) {
+                        // Failed to connect or login.
+                        break;
```

Comment:
I'd change to `return` for better readability.

Reply:
`break` is used everywhere else in this `switch`. I would rather leave it as is for consistency instead of changing all breaks to returns in the switch.